lumigator/frontend/src/components/pages/LDatasets.vue

```diff
@@ -1,58 +1,109 @@
 <template>
-  <div class="l-datasets">
-    <div class="l-datasets__list-container">
-      <ul v-if="datasets && datasets.length"
-          class="l-datasets__list"
-      >
-        <li v-for="dataset in datasets"
-            :key="dataset.id"
-        >
-          <div class="l-datasets__list-card"
-               @click="onDatasetSelect(dataset.id)"
-          >
-            <span> {{ formatDate(dataset.created_at) }}</span>
-            <span>File name: {{ dataset.filename }}</span>
-            <span>Ground truth: {{ dataset.ground_truth ? '✅ ' : ' ❌' }}</span>
-            <span>Size: {{ dataset.size }} kb</span>
-          </div>
-          <span class="l-datasets__list-remove"
-                @click="onRemoveDataset(dataset.id)"
-          >🗑️</span>
-        </li>
-      </ul>
+  <div
+    class="l-datasets"
+    :class="{'is-empty': datasets.length === 0 || !datasets}"
+  >
+    <div
+      v-if="datasets.length > 0"
+      class="l-datasets__header-container"
+    >
+      <l-page-header
+        title="Datasets"
+        subtitle="Excepteur sint  occaecat cupidatat non proident, sunt in culpa qui iest."
```

Comment:
❗ Do we have some copy that will go in here?

Reply:
Not yet

Reply:
Okie dokey. I'm not sure where you 'see' this on the page TBH, but just wanted to call it out in case it was missed accidentally.